editor/components/inserter/test/menu.js

```diff
@@ -90,256 +91,238 @@ const items = [
 	sharedItem,
 ];
 
-describe( 'InserterMenu', () => {
-	// NOTE: Due to https://github.com/airbnb/enzyme/issues/1174, some of the selectors passed through to
-	// wrapper.find have had to be strengthened (and the filterWhere strengthened also), otherwise two
-	// results would be returned even though only one was in the DOM.
+const DEFAULT_PROPS = {
+	position: 'top center',
+	instanceID: 1,
```

Comment:
`withInstanceId` passes a prop called `instanceId`, not `instanceID`. Should we be concerned that this is not failing? Or do we not need it at all?

Reply:
ooo good catch!  I wouldn't expect things to fail in the _current_ tests because `instanceId` is only implemented on the search label `for` attribute and the search input `id` attribute.  We're not testing those rendered attributes specifically.

Still, this should probably be fixed by either removing instanceID from the props or just changing it to `instanceId`.  I would prefer removing it.  I also only think this should be fixed to prevent confusion for other devs looking at this test, but really its not impacting existing tests (similar to how the lack of a `rootUID` passed in as props on tests isn't affecting things. 

Fix in a new pull?

Reply:
Yep, new pull sounds great. If we don't need it, removing it sounds best.

Reply:
Somewhat related.  I wonder if these components should have defaultProps for implemented props that could be undefined if the component isn't wrapped (eg. `instanceId` or `rootUID`?  Or is the component just exported for convenience with testing?

Reply:
>Or is the unwrapped component just exported for convenience with testing?

Primarily, yes.

Reply:
https://github.com/WordPress/gutenberg/pull/7984